tools/aws_lab_setup/README.md

```diff
@@ -1,50 +1,85 @@
 
-#### Email Options ####
+
+## Email Options
 This provisioner by default will send email to participants/students containing information about their lab environment including IPs and credentials. This configuration requires that each participant register for the workshop using their full name and email address.   Alternatively, you can use generic accounts for workshops.  This method offers the advantage of enabling the facilitator to handle "walk-ins" and is a simpler method overall in terms of collecting participant information.
 
 Steps included in this guide will be tagged with __(email)__ to denote it as a step required if you want to use email and __(no email)__ for steps you should follow if you chose not to use email   
 
 **WARNING** Emails are sent _every_ time the playbook is run. To prevent emails from being sent on subsequent runs of the playbook, add `email: no` to `extra_vars.yml`.
 
-#### Lab Configuration ####
-To set up the lab for Ansible training, follow these steps.
+## Lab Setup
+To provision the workshop onto AWS use the following directions:
+
+### One Time Setup
 
 1. Create an Amazon AWS account.
 
-2. Create an ssh key pair called 'ansible'. (To create go Services -> EC2 -> Network & Security -> Key Pairs) Download the private key to your `.ssh` directory, e.g. to `.ssh/ansible.pem`. Alternatively, you can upload your own public key into AWS.
+2. Create an Access Key ID and Secret Access Key.  Save the ID and key for later.
 
-      If using an AWS generated key add it to the ssh-agent:
+  - New to AWS and not sure what this step means?  [Click here](aws-directions/AWSHELP.md)
 
-        ssh-add ~/.ssh/ansible.pem
+3. Install `boto` and `boto3`.
 
-3. Create an [Access Key ID and Secret Access Key](http://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) (you should be using IAM and [not your AWS account directly](http://docs.aws.amazon.com/general/latest/gr/managing-aws-access-keys.html)).  Save the ID and key for later.
+        pip install boto boto3
 
-4. Create Amazon VPC.   Use the wizard and just accept the defaults.   It should create a VPC and a subnet. Save this info for later.
+4. Create an [Access Key ID and Secret Access Key](http://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) (you should be using IAM and [not your AWS account directly](http://docs.aws.amazon.com/general/latest/gr/managing-aws-access-keys.html)).  Save the ID and key for later.
 
-5. Install `boto` and `boto3`.
+5. Install the `passlib` library
 
-        pip install boto boto3
+        pip install passlib
 
-6. Create a `boto` configuration file containing your AWS access key ID and secret access key.
+6. Clone the lightbulb repo:
```

Comment:
If you merge this into the lightbulb repo, this line no longer makes sense.

Reply:
can you elaborate?  Originally I thought I must have a bad link back to network-automation/lightbulb but isn't this the right link-> `https://github.com/ansible/lightbulb.git`

Reply:
Because upon merge, these instructions will be *in* the lightbulb repo, which is already cloned if you're using this, right?

Reply:
I guess I am making some assumptions here based on how a user would first grab this.  I often send the link to the instructions on github.com, so I was assuming they were reading them from the website.  I also think a lot of network engineers have limited experience with git altogether so I literally put that step in.  I am fine with ripping this out if you think its too verbose or confusing.

Reply:
Maybe just add "if you haven't done so already" to clarify.